src/serac/numerics/mesh_utils.hpp

```diff
@@ -205,7 +205,7 @@ struct InputOptions {
    * @brief The mesh input options (either file or generated)
    *
    */
-  std::variant<FileInputOptions, GenerateInputOptions> extra_options;
+  std::variant<FileInputOptions, BoxInputOptions, NBallInputOptions> extra_options;
```

Comment:
So in the future, we might have `RingInputOptions` and etc?

Reply:
Yep, the idea was that those could be added in as we need them (for tests etc)